PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
-#### What does this PR do?
+#### What changes are we making? Why are we making them?
```

Comment:
From the peanut gallery: "What changes are we making? What problem do these changes solve? Why solve the problem this way?"

Reply:
maybe: What changes are we making? → 

> ### How are we solving the problem? (Explain the changes.)
> ...

Reply:
I think "What changes are we making?" could be taken too literally.

"What technical or business problems does this PR solve?" might be all that's needed IMO. That too me is the crux of what this section is asking.

Reply:
how about:

> #### What technical/business problem(s) are we solving?
> ...
>
> #### How are we solving it? (Explain the changes and the rationale)
> ...
>